test/integration/contract-aci.js

```diff
@@ -520,7 +543,7 @@ describe('Contract instance', function () {
 
       it('Invalid arguments length', async () => {
         await expect(testContract.methods.datTypeGFn())
-          .to.be.rejectedWith('Non matching number of arguments. Got 0 but expected from 1 to 1')
+          .to.be.rejectedWith('Non matching number of arguments. datTypeGFn expects between 1 and 1 number of arguments but got 0')
```

Comment:
weird error message

Reply:
@davidyuk does this message come from calldata-lib ? just wondering